File: Training JS #32: methods of Math---round() ceil() and floor().js
function roundIt(n){
  let v = String(n).split(".")
  if(v[0].length > v[1].length){
    return Math.floor(n)
  }else if(v[0].length < v[1].length){
    return Math.ceil(n)
  }else{
    return Math.round(n)
  }
  
}

File: Training JS #32: methods of Math---round() ceil() and floor()/Training JS #32: methods of Math---round() ceil() and floor().py
import math
def round_it(n):
    n_c = str(n).split(".")
    if len(n_c[0])<len(n_c[1]):
        return math.ceil(n)
    elif len(n_c[0])>len(n_c[1]):
        return math.floor(n)
    else:
        return round(n)
